packages/notebook/src/browser/libro/editor/opensumi-editor.ts

```
@@ -0,0 +1,826 @@
+    this.host = options.host;
+    this.host.classList.add('libro-opensumi-editor-container');
+    this._uuid = options.uuid || uuid();
+
+    this._model = options.model;
+
+    const config = options.config || {};
+    const fullConfig = (this._config = {
+      ...libroOpensumiEditorDefaultConfig,
+      ...config,
+      mimetype: options.model.mimeType,
+    });
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)